R/remove_stop_codon.R

```diff
@@ -0,0 +1,13 @@
+#' Remove stop codon at end of sequence
+#'
+#' Stop codons at the end of the amino acid sequences are removed
+#'
+#'
+#' @param faa_df A dataframe containing two columns: the sequence name and amino acid sequence
+#'
+#' @return The input dataframe without the stop codons at the end of sequences
+
+remove_stop_codon <- function(faa_df) {
+  data.frame(lapply(faa_df, function(x) gsub("\\*$", "", x)), stringsAsFactors = FALSE)
```

Comment:
@Legana I think you should use vectorised gsub here instead of `lapply`.  It will be much faster.

Reply:
@iracooke done :) please see 7eb0732d05e8f53f05d18fd23e34f1b8fb6e7f46 for issue #13